service/src/main/java/bio/terra/workspace/service/workspace/flight/create/workspace/WorkspaceCreateFlight.java

```diff
@@ -64,6 +65,14 @@ public WorkspaceCreateFlight(FlightMap inputParameters, Object applicationContex
               new CreateWorkspacePoliciesStep(
                   workspace, policyInputs, appContext.getTpsApiDispatch(), userRequest),
               serviceRetryRule);
+
+          addStep(
```

Comment:
This will link instead of merge. And it is good to do it before the cloning merge below.